### ISSUE TITLE
Question:

### ISSUE THREAD

**Issue by CsharpNoobDude** [Evrey/SHC_Campaign#1]
I just want to put in some thoughts:
-Will you create extra AIV's that are suited EXACTLY to the maps you created? For example, towers on hills of the map, or only walls there they are really needed, Sultan building hunter huts for example because you added deer. Or the rat building iron because there is one on the map. could all be done with the AIV editor. 
-More thoughts will be added whenever I have them :D

But all in all, I think this is more than a brilliant idea. And don't get me wrong, I love your aiv files, but imo it would be just awesome if it would seem as if the ai is actually really using the territory 100%

**Comment by Evrey** [comment]
> Will you create extra AIV's that are suited EXACTLY to the maps you created?
>
> [...] if it would seem as if the ai is actually really using the territory 100%

Nope, I will use my SHC_AIV mod as it is. No special treatment. Instead, I'm optimising the maps towards the specific AI castles I picked. I already have plans for later missions, in one of which I'll use Philipp to block the only passage to the other enemies, using his Maginot Line wall. A later iteration of that mission idea will use the Wolf to block the only passage, using his whole castle's width.

> For example, towers on hills of the map, [...]

I'd put those things right in the editor, instead of modifying the AI castles. I want to keep the original skirmish campaign feeling this way. To add more spice to the missions, I'll disable some buildings depending on the mission. I have an Acre mission planned, where you are actually really put between a city Acre and Saladin.

> More thoughts will be added whenever I have them :D

Sure, keep 'em coming!

**Comment by CsharpNoobDude** [comment]
@Evrey Thanks for replying.
Your ideas sound cool. But, would you be so kind to add a "fan missions" folder maybe? Because I may want to upload something I had in my. With my own aiv (optimised after your 1 hour+ plus video, still holy shit dude that was just so kind of you) integrated as well as a basic map :D
Have you seen my "Udwin AIV"? I started looking for cool castles udwin designed in his videos, and I've already got one uploaded, and one in progress.
Another question, not regarding this repository though.
I have send you a private message via youtube, did you get that?

Edit:

> I'd put those things right in the editor, instead of modifying the AI castles. I want to keep the original  > skirmish campaign feeling this way. To add more spice to the missions, I'll disable some buildings       > depending on the mission. I have an Acre mission planned, where you are actually really put between a city Acre and Saladin.

If you place units there, the ai will just take them and place them on their keep.

**Comment by Heroesflorian** [comment]
> If you place units there, the ai will just take them and place them on their keep. 

Depending on how it is done, not necessarily. Either he could place troops (of types that the respective AI have troop spots placed in their AIV) on the map, then the AI will treat those like additional starting troops and distribute them accordingly. Or he could place them on the map after converting it to a skirmish-scenario, that would work as well. 
But either way, I think what Evrey meant there, was that he would tend to rather make the map feature a hill where his castles would have a tower anyway, instead of adding an extra tower to his castles to fit the hill of the map - or for short, landscape follows castle instead of the other way round.
